mitodl/open-discussions
# Elasticsearch 7.10 client only (PR 1 of 3)

#### Pre-Flight checklist


#### What are the relevant tickets?
updates https://github.com/mitodl/open-discussions/issues/3494

#### What's this PR do?
This PR is a reversion + inclusion of a different approach to handle the different mapping structure in 6 for upgrade to 7 based on Elasticsearch's suggestion.

#### How should this be manually tested?
Pull master with 6.8. Generate data + indices then pull this branch and attempt a reindex. This should generate errors. You may still get deprecation warnings. This is expected and will be handled after 7.10 upgrade
